member-service/src/main/java/com/hedvig/memberservice/aggregates/MemberAggregate.kt

```diff
@@ -305,43 +305,45 @@ class MemberAggregate() {
     }
 
     private fun applyMemberIdentifiedEventFromZignSecSignCommand(cmd: ZignSecSignCommand) {
+        val request = objectMapper.readValue(cmd.provideJsonResponse, ZignSecNotificationRequest::class.java)
+        val idProvider = request.identity?.idProviderName ?: return
+        if (member.firstName == cmd.firstName &&
+            member.lastName == cmd.lastName &&
+            member.ssn == cmd.personalNumber) return
```

Comment:
Depending on how we treat the "MemberIdentifiedEvent" concept, maybe we can have another approach here. The code is a bit "axon-y" I could maybe elaborate a bit more over CoScreen what I mean if it's cryptic 😄
```suggestion
        if (isIdentified) return
```